contributed_definitions/nyaml/NXapm_paraprobe_distancer_config.yaml

```diff
@@ -142,16 +148,223 @@ NXapm_paraprobe_distancer_config(NXobject):
           exists: optional
           method(NX_CHAR):
           match(NX_NUMBER):
+    
     # point_set_to_polyline_set(NXapm_paraprobe_tool_config):
-    common(NXapm_paraprobe_tool_common): 
+    common(NXapm_paraprobe_tool_common):
       status(NX_CHAR):
       programID(NXprogram):
-        exists: [min, 1, max, infty]
+        exists: ['min', '1', 'max', 'unbounded']
```

Comment:
the old line looked simpler. was change needed, because it was incorrect?  
Note that [1,INF] or [1,unbounded] would be even simpler to write.